main.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"fmt"
+	"github.com/BuxOrg/go-buxclient/xpriv"
+	"os"
+)
+
+func main() {
+
+	keys, err := xpriv.Generate()
+
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	xpubKeyName := "xpub_key.txt"
+	xprvKeyName := "xprv_key.txt"
+
+	xpubKeyFile, err := os.Create(xpubKeyName)
+
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	defer func(xpubKeyFile *os.File) {
+		err := xpubKeyFile.Close()
+		if err != nil {
+			fmt.Println(err)
+		}
+	}(xpubKeyFile)
+
+	_, err = xpubKeyFile.WriteString(keys.XPub().String())
+
+	if err != nil {
+		fmt.Println(err)
```

Comment:
same here. it should stop the execution if there is an error

Reply:
Done